packages/design-system/src/components/contextMenu/menuItem.js

```diff
@@ -143,7 +139,6 @@ export const MenuItem = ({
     return (
       <Button
         ref={itemRef}
-        aria-label={itemLabel}
```

Comment:
Given the set up of context menus that don't have keyboard shortcuts,  could we keep the aria-labels for links and buttons that have no shortcuts?

Reply:
yup, I'll add that back in 👍 good catch